java/examples/build.gradle

```diff
@@ -0,0 +1,25 @@
+plugins {
```

Comment:
examples should be under the `${root}/examples` folder, not under the `java` folder. This reduces the user's ability to find the app.

Reply:
This adds difficulties on dependency resolution (line 12), until java client released to maven.
But I think it is possible to commit a directory symlink. @acarbonetto can you try this?

Reply:
I will try.  
Ultimately, we want to be able to create a distribution JAR, and at that point a separate gradle project may be fine.